app/src/main/java/com/dscout/membranevideoroomdemo/RoomActivity.kt

```diff
@@ -221,28 +200,72 @@ fun ParticipantCard(
     size: Size,
     onClick: (() -> Unit)? = null
 ) {
-    Box(
-        modifier = Modifier.clickable(
-            indication = null,
-            interactionSource = remember { MutableInteractionSource() }
-        ) {
-            onClick?.invoke()
+    fun shouldShowIcon(trackType: String): Boolean {
+        return when (trackType) {
+            "audio" -> {
+                participant.audioTrack == null || (
+                    participant.tracksMetadata.isNotEmpty() &&
+                        (participant.tracksMetadata[participant.audioTrack.id()]?.get("active") as? Boolean) != true
```

Comment:
can we make it shorter or divide it into shorter statements? a three line check doesn't look too good

Reply:
added a new helper function